homeassistant/components/lifx/services.yaml

```diff
@@ -0,0 +1 @@
+# LIFX services are defined in the light platform.

```

Comment:
If the custom services are registered under the light domain they should instead be registered under the lifx domain and then we can populate this file with the service descriptions.

Do you own a lifx device?

Reply:
@MartinHjelmare Do you mean we prefer (or must) call custom services with `lifx.custom_service_by_lifx` than `light.custom_service_by_lifx`? Even if the second allows to find easily all the services available for a device entity?

Reply:
We don't allow adding new custom services to the platform domain. They must be added to the integration domain.

So we shouldn't add new code to the existing integrations that is against our standards. Better to change the old code while we're changing things.

Reply:
@MartinHjelmare I thought the goal of #27289 is to document the existing services, not to immediately re-architect every platform to conform to new design standards. If the plan is to change all existing non-conforming platforms, can we open a new issue against the offending platforms as part of a different dev push? 
I do own many LIFX devices and test so that's not an issue.

Reply:
Perhaps we can do it in two steps. First identify where it's a missing documentation and where we should review the code. 
@MartinHjelmare is there any documentation somewhere with this architecture constraints explained ? It could be a good starting point if we want to start an issue to follow the updates needed?

Reply:
I think it's ok to skip the integrations that have services registered under the wrong domain when we add the service descriptions.

It's a good suggestion to identity which integrations need what changes and update lists.